src/tests/tribol_coupling_scheme.cpp

```diff
@@ -594,8 +594,8 @@ TEST_F( CouplingSchemeTest, invalid_mesh_in_coupling_scheme )
   int numFaces2 = 1;
   constexpr int numTotalNodes1 = 2;
```

Comment:
Can you clarify when using int is ok, and when we should use tribol::IndexT? I can see myself developing and easily breaking a host-code built or ability to run on the gpu.